cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -36,19 +43,25 @@ def test_single_datastore_single_vm_migration(request, appliance, v2v_providers,
                                             conversion_tags,
                                             form_data_vm_obj_single_datastore):
     infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    # form_data_vm_obj_single_datastore has form_data at [0]
-    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore[0])
+    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
 
     @request.addfinalizer
     def _cleanup():
         infrastructure_mapping_collection.delete(mapping)
 
+    src_vm_obj = form_data_vm_obj_single_datastore.vm_obj[0]
```

Comment:
what is the 0th index here? Can you put a comment that vm_obj is a list and you are getting the 0th element because.

Reply:
Added